examples/larger-examples/datalog/connect-graph.flix

```diff
@@ -46,7 +46,7 @@ pub def connectedComponentRep(g: Graph[node]): #{ ComponentRep(node, node) | r }
         // `rep` is the highest node of the component.
         ComponentRep(n, rep) :- Reachable(n, rep), not ReachUp(rep).
     };
-    solve nodes, edges, reachability project ComponentRep
+    solve nodes, edges, reachability inject ComponentRep
```

Comment:
shouldn't solve ... project ... still use project?

Reply:
We renamed project to inject.

Reply:
Gosh. You are right

Reply:
@jaschdoc As Jonathan points out. For solve it should still be `solve ... project ...`.

Reply:
Ah ok I see that makes more sense.

Reply:
Its a question of direction.